data/strings/strings.txt

```diff
@@ -28803,3 +28803,48 @@
     vi = Âm lượng
     zh-Hans = 音量
     zh-Hant = 音量
+
+  [bookmarks_export]
+    tags = android
+    comment = Bookmark categories screen, button that opens folder selection dialog to export KML/KMZ/GPX/KMB files
```

Comment:
Why it should open a folder selection dialog? I would expect to see a standard system sharing dialog where users can choose the app where to export the file.

Reply:
It is a copy-paste issue in comment, it will be a share dialog